src/block/icon-list/style.scss

```diff
@@ -1,10 +1,10 @@
 .stk-block-icon-list {
-	ol > li {
-		list-style-image: none !important;
+	ol {
+		list-style-image: none;
 	}
 
 	// Line height creates unnecessary row gaps.
-	ul > li::marker {
+	::marker {
 		line-height: 0;
 	}
 }
```

Comment:
@blankart we can simplify this further, we shouldn't use `!important` also. We need to be more conscious about the styles which will be loaded in the frontend.